MonoGame.Framework.Content.Pipeline/TextureImporter.cs

```diff
@@ -198,7 +198,7 @@ TextureContent LoadImage(string filename)
             int width, height, comp;
             byte[] data = null;
             using (var stream = File.Open(filename, FileMode.Open, FileAccess.Read, FileShare.Read))
-                data = reader.Read(stream, out width, out height, out comp, Imaging.STBI_rgb_alpha);
+                data = ImageReader.Read(stream, out width, out height, out comp, Imaging.STBI_rgb_alpha);
```

Comment:
You don't need the reader instance anymore when loading an image in the changed files.  Sorry, I missed that before.

Reply:
Thanks! I've made ImageReader static and removed all places it had been instantiated.